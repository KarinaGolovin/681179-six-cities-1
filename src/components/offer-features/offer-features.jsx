import React from 'react';
import PropTypes from 'prop-types';

export const OfferFeatures = ({featureList}) => {
  return (
    <div className="property__inside">
      <h2 className="property__inside-title">What&apos;s inside</h2>
      <ul className="property__inside-list">
        {featureList.map((feature) => {
          return (
            <li className="property__inside-item" key={feature}>
              {feature}
            </li>
          );
        })}
      </ul>
    </div>
  );
}

OfferFeatures.propTypes = {
  featureList: PropTypes.array,
}
