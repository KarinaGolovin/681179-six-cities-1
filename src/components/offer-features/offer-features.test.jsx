import React from 'react';
import renderer from 'react-test-renderer';
import {OfferFeatures} from './offer-features.jsx';

const mock = [`Test`, `Test1`, `Test2`];

it(`OfferFeatures correctly renders`, () => {
  const component = renderer.create(
      <OfferFeatures
        featureList={mock}
      />
  ).toJSON();

  expect(component).toMatchSnapshot();
});
