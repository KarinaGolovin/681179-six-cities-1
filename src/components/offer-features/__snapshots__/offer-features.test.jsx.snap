// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OfferFeatures correctly renders 1`] = `
<div
  className="property__inside"
>
  <h2
    className="property__inside-title"
  >
    What's inside
  </h2>
  <ul
    className="property__inside-list"
  >
    <li
      className="property__inside-item"
    >
      Test
    </li>
    <li
      className="property__inside-item"
    >
      Test1
    </li>
    <li
      className="property__inside-item"
    >
      Test2
    </li>
  </ul>
</div>
`;
